pages/monbar.php

```diff
@@ -73,15 +76,96 @@
         <button>
             <img id="btnAbout" src="../ressources/images/info.svg" alt="Question Mark" width="24" height="24">
         </button>
-        <button>
+        <button type="button" id="publish" data-bs-toggle="modal" data-bs-target="#monModal">
             <img id="btnPublish" src="../ressources/images/feather.svg" alt="Feather" width="24" height="24">
         </button>
+
     </aside>
 
+    <!-- Le modal -->
+    <div id="monModal" class="modal fade">
+        <div class="modal-dialog modal-xl">
+            <!-- Contenu du modal -->
+            <div id="publication-content" class="modal-content">
+                <div class="modal-header">
+                    <h4 class="modal-title">Créer un cocktail</h4>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal">&times;</button>
+                    <!-- symbole x pour fermer -->
+                </div>
+
+                <div class="modal-body" id="page-publication">
+
+                    <form>
+                        <label for="nom-cocktail">Nom du cocktail:</label><br>
+                        <textarea id="name-texte" name="nom-cocktail" rows="1" cols="50"></textarea><br>
+
+                        <label for="ingredient-cocktail">Ingrédients:</label><br>
+                        <div id="ingredient-input">
+                            <div id="add-form">
+                                <input type="text" id="ingredient-nom" placeholder="Nom de l'ingrédient...">
+                                <input type="text" id="ingredient-quantite" placeholder="Quantité...">
+                                <input type="text" id="ingredient-unit" placeholder="Unité...">
+
+                                <button type="button" class="button-publish" id="ajouter-ingredient">
+                                    <img class="btn-icon" src="../ressources/images/plus.svg" alt="+">
```

Comment:
L'attribut `alt` doit décrire l'objectif ou la fonction du bouton plutôt que l'image elle-même.
```suggestion
                                    <img class="btn-icon" src="../ressources/images/plus.svg" alt="Ajouter un ingrédient">
```